README.md

```diff
@@ -3,17 +3,18 @@
 # Portfolio - Mobile skeleton app
 
 > Module 1 portfolio project, skeleton app page with no functionality.
+> The mobile page should be a clone of [this template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A27)
 
-<!--![screenshot](./app_screenshot.png)-->
+Here you can see a picture ---> [screenshot](./app_screenshot.png)
 
 <!--Additional description about the project and its features.-->
 
 ## Built With
 - css
 - html
 
-<!--## Live Demo-->
-
+## Live Demo
+https://gabrielyea.github.io/module1-portfolio-setup/
```

Comment:
- [ ] **OPTIONAL**
It's more clean and good-looking if you use the markup language to shorten the links.
Like so!
[Live Demo Link](https://gabrielyea.github.io/module1-portfolio-setup/)